api/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "setup": "npm run setup:install && npm run setup:db",
+    "setup:install": "npm install",
+    "setup:db": "npm run db:migrate && npm run db:seed",
+    "db:migrate": "./node_modules/.bin/sequelize db:migrate",
+    "db:seed": "./node_modules/.bin/sequelize db:seed:all",
+    "db:undo": "./node_modules/.bin/sequelize db:migrate:undo",
+    "db:seed:undo": "./node_modules/.bin/sequelize db:seed:undo",
+    "db:migrate:undo": "./node_modules/.bin/sequelize db:migrate:undo",
```

Comment:
Added these so you can run `npm run COMMANDS IN THE COMMENT`

Reply:
oooo thats a script